scenes/quests/lore_quests/quest_001/1_music_puzzle/components/dialogues/musician.dialogue

```diff
@@ -37,70 +36,75 @@ Musician: Hello again, StoryWeaver. Would you like to hear about the stones agai
+	Musician: Look at the symbol on the second altar to see if you can work out what the next part of the melody might be.
 	do advance_hint_level()
 	=> END
 if get_limited_hint_level() == 1:
-	Musician: Someone around may know more about the signs. You may even find them somewhere else.
+	Musician: The symbol seems to suggest that you need to move up one note, and repeat the first melody.
 	do advance_hint_level()
 	=> END
 if get_limited_hint_level() == 2:
-	Musician: There's the Luthier in the mountain region past the river. I know he studied the signs for a while.
+	Musician: I think that altar symbol means start with the second note... then third, fourth, and sixth.
 	do advance_hint_level()
 	=> END
 
 ~ hint_melody_2
 if get_limited_hint_level() == 0:
-	Musician: This other sign is similar, and yet... it feels the opposite.
+	Musician: For the third part of the melody, the altar symbol suggests the next melody is played backwards.
 	do advance_hint_level()
 	=> END
 if get_limited_hint_level() == 1:
-	Musician: The forest hides secrets that may help you figure this out.
+	Musician: Try playing the previous melody but in reverse.
 	do advance_hint_level()
 	=> END
 if get_limited_hint_level() == 2:
-	Musician: There's a glade nearby. The Minstrel lives there. He can help you.
+	Musician: Ok, I think it is sixth, fourth, third, and second for this one.
 	do advance_hint_level()
 	=> END
 
 ~ hint_melody_3
 if get_limited_hint_level() == 0:
-	Musician: I don't recall anything more... But you are so close! I know you can do this.
+	Musician: Last one... you can do this! Look at the symbol and play the final melody.
 	do advance_hint_level()
 	=> END
 if get_limited_hint_level() == 1:
-	Musician: If you reached this far, you know what comes next.
+	Musician: Think about what you have already played... the answer is there.
 	do advance_hint_level()
 	=> END
 if get_limited_hint_level() == 2:
-	Musician: The Minstrel in the glade nearby may help you, if you haven't seen him yet.
+	Musician: Play the very first melody, but in reverse. Fifth, third, second, and first should do it.
 	do advance_hint_level()
 	=> END
 
-~ well_done
-Musician: The melody... it's not just music. It opens the portal to the Ink Well. Let's go find more ink so songs and stories can be written again.
-Musician: But we must be careful, the InkDrinkers dwell there!
+~ item_hint
+Musician: You did it!... That's the magical melody... how wonderful. It's been so long since I heard it.
+Musician: You have opened the portal to the Ink Well. Let's go there to collect ink so songs and stories can be written again.
+Musician: But we must be careful, InkDrinkers dwell there!
+Musician: The melody awakened the memory of song. Collect the Memory threads before you go!
 => END
 
-~ item_hint
-Musician: (PLACEHOLDER) You did it! The melody worked. I left a special item near the stones, grab it before you go!
+~ well_done
+StoryWeaver: Ink Well, here we come!
 => END
```

Comment:
There is no guarantee that the player interacts with the musician after completing the puzzle but before interacting with the thread. If the player goes straight to the thread, they will miss all the exposition about the Ink Well.

We can address this later – the trick might be to play the musician's dialogue automatically when the puzzle is solved.